cardinal/ecs/engine.go

```diff
@@ -318,6 +318,47 @@ func (e *Engine) RegisterMessages(txs ...message.Message) error {
 	return nil
 }
 
+func (e *Engine) registerInternalQueries() {
+	signerQueryType, err := NewQueryType[QueryPersonaSignerRequest, QueryPersonaSignerResponse](
+		"signer",
+		querySigner,
+		WithCustomQueryGroup[QueryPersonaSignerRequest, QueryPersonaSignerResponse]("persona"),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	debugQueryType, err := NewQueryType[DebugRequest, DebugStateResponse](
+		"debug",
+		queryDebugState,
+		WithCustomQueryGroup[DebugRequest, DebugStateResponse]("debug"),
+	)
```

Comment:
this will show up as `query/debug/debug`. do we want to change it to `query/debug/state`?